src/main/java/org/matsim/episim/InfectionEventHandler.java

```diff
@@ -608,15 +608,14 @@ private EpisimFacility createHomeFacility(EpisimPerson person) {
 		return new EpisimPerson.Activity(actType, episimConfig.selectInfectionParams(actType));
 	}
 
-	private Id<ActivityFacility> createEpisimFacilityId(HasFacilityId event) {
+	static Id<ActivityFacility> createEpisimFacilityId(HasFacilityId event, EpisimConfigGroup episimConfig) {
```

Comment:
Is this method now used anywhere else? I think it could be moved to the reply handler entirely.

Reply:
It's still used in the init method of the InfectionEventHandler, and I'm not sure if I can remove them there with event.getFacilityId() in all use cases. In the case that this is possible, the function could be moved.

Reply:
Probably you should also check if the interfaces specific to the EventsManger can be removed. I think it needs to be made clear that it can not be used with the  EventHandler anymore, but the use of the ReplayHandler is absolutely required. 
I guess this is the case now if filtering and processing is performed there.